src/Hud/Menu/MenuPlugin.cs

```diff
@@ -201,12 +201,12 @@ private void CreateMenu()
             MenuItem xpRateMenu = AddChild(MenuRootButton, "Xph & area", settingsHub.XpRateSettings.Enable, "F10");
             MenuItem areaName = AddChild(xpRateMenu, "Only area name", settingsHub.XpRateSettings.OnlyAreaName);
             AddChild(areaName, "Show latency", settingsHub.XpRateSettings.ShowLatency);
-            AddChild(areaName, "Latency color", settingsHub.XpRateSettings.LatencyTextColor);
+            AddChild(areaName, "Show fps", settingsHub.XpRateSettings.ShowFps);
```

Comment:
Ok, it doesn't really bother me either way just seemed unnecessary, I reverted the AreaColor changes @TehCheat was already doing what I mentioned I just didn't realize you had to change zones for it to update